tests/test_hear_cleanup.py

```
@@ -95,25 +140,32 @@ def test_cleanup_on_exit(self):
-			p.join(self.FINAL_JOIN_TIMEOUT)
+			p.join(self.PROCESS_TIMEOUT_SECONDS)
 			self.assertIsNotNone(p.exitcode)
-			self.assertEqual(int(p.exitcode), int(0))
-			theResult = (int(p.exitcode) <= int(0))
+			self.assertEqual(
+				int(p.exitcode),
+				int(self.EXPECTED_STOP_EXIT_CODE),
+				"CEP-8 VIOLATION.",
+			)
+			theResult = (int(p.exitcode) <= int(self.EXPECTED_STOP_EXIT_CODE))
 		except Exception as err:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)